libs/blocks/gnav/gnav.css

```diff
@@ -645,11 +645,19 @@ nav.gnav .gnav-navitem-menu .gnav-promo.card img {
 
 /* PROFILE */
 
+.gnav:not(.signed-in) .gnav-profile {
+  min-width: 92.25px;
```

Comment:
For accessability reasons , I'd recommend always using rem or em over pixels. Here are some great articles which explain why: 
* https://engageinteractive.co.uk/blog/em-vs-rem-vs-px
* https://zellwk.com/blog/media-query-units/

This comment goes for all the px entries

Reply:
Updated